lib/bosh_deployment_resource/bosh_release.rb

```diff
@@ -29,7 +29,7 @@ def manifest_file_contents
 
       Archive::Tar::Minitar::Reader.open(tgz) do |reader|
         reader.each_entry do |entry|
-          next unless File.basename(entry.full_name) == "release.MF"
+          next unless File.basename(entry.full_name) == "release.MF" && entry.full_name.match('PaxHeader/release.MF').nil?
```

Comment:
Ah good catch. I suppose the point of `basename` really just to strip out a `./` if present. Perhaps instead we could do something like this?:

```ruby
next unless Pathname.new(entry.full_name).cleanpath.to_s == "release.MF"
```

That should be a bit more airtight and then we don't need this seedy little detail. :)